src/schema/v2/homeView/sections/GalleriesNearYou.ts

```diff
@@ -19,4 +20,14 @@ export const GalleriesNearYou: HomeViewSection = {
   },
   ownerType: OwnerType.galleriesForYou,
   requiresAuthentication: false,
+
+  resolver: withHomeViewTimeout(async (_parent, _args, _context, _info) => {
+    return {
+      title: "Galleries Near You",
+      subtitle: "Follow these local galleries for updates on artists you love.",
+      href: "/galleries-for-you",
```

Comment:
**question**: I'm noticing that we don't have an equivalent [web route with this path](https://www.artsy.net/galleries-for-you) and I know we've talked recently about only using stable / widely-supported / "canonical" paths in these `href` attributes.

Would `entityType: OwnerType.galleriesForYou` be a better option in this case?

Reply:
Ohh, good catch. I will take a look at that.

Reply:
Do you have a preference between any of these options @dblandin ?

```ts
// option A - use the entity type, as suggested above
{
  entityType: OwnerType.galleriesForYou
}
```

```ts
// option B - use the entity ID, since this uniquely identifies a resource
{
  entityID: OwnerType.galleriesForYou
}
```


```ts
// option C - use both, for added explicitness
{
  entityType: "Page" // or something equivalent from Cohesion?
  entityID: OwnerType.galleriesForYou
}
```

I opted for **Option C**…

-  (MP) https://github.com/artsy/metaphysics/pull/6116/commits/f7ea23f849c3d8d40aa0bdd5b1f3addccd18600d
- (Eigen) https://github.com/artsy/eigen/pull/10922/commits/92fbc3ffd7753ffb38ad189068d2528353a64647

…but happy to change as well.

Reply:
Yeah, let's roll with (C). Looks good 👍🏼